x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiPageContent,
+} from '@elastic/eui';
```

Comment:
eslint will probably error now that we're no longer using `EuiPageHeaderSection` and `EuiTitle`

```suggestion
import { EuiPageHeader, EuiPageContentBody, EuiPageContent } from '@elastic/eui';
```